lib/improver/ensemble_copula_coupling/ensemble_copula_coupling.py

```diff
@@ -442,7 +442,8 @@ def _probabilities_to_percentiles(
         prob_slices = np.around(prob_slices, 9)
 
         # Invert probabilities for data thresholded above thresholds.
-        relation = forecast_probabilities.attributes['relative_to_threshold']
+        relation = forecast_probabilities.coord(
+            var_name="threshold").attributes['spp__relative_to_threshold']
```

Comment:
For stuff outside the unit tests, it would be good to keep things backward compatible I think, to make sure we don't run into any issues with science trials / tests.  So could you use the `improver.utilities.cube_checker.find_threshold_coordinate` function for this (and for other non-test modules) please?